=0.11.0.dev20170322155718

```diff
@@ -0,0 +1,6 @@
+Collecting coala
+  Downloading coala-0.10.0.tar.gz (2.0MB)
+    Complete output from command python setup.py egg_info:
+    coala supports only python 3.4 or later.
+    
```

Comment:
Not quite the way that you want to do it, don't use github UI for anything. Just change it locally and remove your other commit. Check the docs.